java/connector-node/risingwave-sink-jdbc/src/main/java/com/risingwave/connector/JDBCSink.java

```diff
@@ -241,16 +247,17 @@ class JdbcStatements implements AutoCloseable {
         public JdbcStatements(Connection conn, int queryTimeoutSecs) throws SQLException {
             this.queryTimeoutSecs = queryTimeoutSecs;
             this.conn = conn;
+            // Set database and schema for Snowflake connections
+            if (config.getJdbcUrl().toLowerCase().startsWith("jdbc:snowflake")) {
+                setSnowflakeDatabaseAndSchema();
+            }
```

Comment:
Given that we already have BatchAppendOnlyJDBCSink, the changes here are no longer needed. I will push a commit to remove it in your branch.